app/jobs/alert_email/base.rb

```diff
@@ -0,0 +1,19 @@
+class AlertEmail::Base < ApplicationJob
+  def perform
+    return unless Rails.env.production?
+
+    subscriptions.each do |s|
+      next if s.alert_run_today?
+
+      vacancies = vacancies_for_subscription(s)
+      next if vacancies.blank?
+
+      AlertMailer.alert(s.id, vacancies.pluck(:id)).deliver_later(queue: :email_alerts)
```

Comment:
`s` is hard to read - cognitive overload. I prefer `subscription`.
It's like when as a new joiner to the team I join a TVS meeting with the DM, PM, UX, and PA about MAT/DSI issues.

Reply:
It wasn't your original code I realise